src/Umbraco.Web.UI.Client/src/views/documenttypes/edit.controller.js

```diff
@@ -307,17 +275,15 @@
             var initialViewSetFromRouteParams = false;
             var view = $routeParams.view;
             if (view) {
-                var viewPath = "views/documenttypes/views/" + view + "/" + view + ".html";
                 for (var i = 0; i < vm.page.navigation.length; i++) {
-                    if (vm.page.navigation[i].view === viewPath) {
+                    if (vm.page.navigation[i].alias.toLowerCase() === view.toLowerCase()) {
```

Comment:
I wanted to compare the alias and the view ignoring the casing. Alias and View are both something that the user themselves can set. Alias can be set by an user when creating their own content app and the view can have casing too because it's from the url.
Alias itself should never be undefined or null because then you have an invalid content app (I don't know if Umbraco itself already catches this) and the view undefined/null is already checked a bit more up.
But is there a better way to check two strings in JS/Angular ignoring casing? Then I can replace it with that

Reply:
You can do `localeCompare` like here: https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Web.UI.Client/src/routes.js#L43

Reply:
That works perfectly! Thanks! Updated the code to use localCompare now